src/rp2_common/pico_cyw43_driver/BUILD.bazel

```diff
@@ -16,35 +16,56 @@ cc_library(
 cc_library(
     name = "pico_cyw43_driver",
     srcs = [
-        "btstack_chipset_cyw43.c",
-        "btstack_cyw43.c",
-        "btstack_hci_transport_cyw43.c",
         "cyw43_bus_pio_spi.c",
         "cyw43_driver.c",
     ],
     hdrs = [
-        "include/pico/btstack_chipset_cyw43.h",
-        "include/pico/btstack_cyw43.h",
-        "include/pico/btstack_hci_transport_cyw43.h",
         "include/pico/cyw43_driver.h",
     ],
     includes = ["include"],
     target_compatible_with = compatible_with_pico_w(),
     deps = [
         ":cyw43_bus_pio",
         ":cyw43_configport",
-        "//bazel/config:PICO_BTSTACK_CONFIG",
         "//src/rp2_common:pico_platform",
         "//src/rp2_common/hardware_clocks",
         "//src/rp2_common/hardware_dma",
         "//src/rp2_common/hardware_irq",
         "//src/rp2_common/hardware_pio",
         "//src/rp2_common/hardware_sync",
         "//src/rp2_common/pico_async_context",
+        "//src/rp2_common/pico_unique_id",
+        "@cyw43-driver//:cyw43_driver",
+    ] +
+    # Only depend on the btstack if its configuration is set up.
+    select({
+        "//bazel/constraint:pico_btstack_config_unset": [],
+        "//conditions:default": [":pico_cyw43_btstack"],
+    }),
+)
+
+cc_library(
+    name = "pico_cyw43_btstack",
+    srcs = [
+        "btstack_chipset_cyw43.c",
+        "btstack_cyw43.c",
+        "btstack_hci_transport_cyw43.c",
+    ],
+    hdrs = [
+        "include/pico/btstack_chipset_cyw43.h",
+        "include/pico/btstack_cyw43.h",
+        "include/pico/btstack_hci_transport_cyw43.h",
+    ],
+    defines = ["CYW43_ENABLE_BLUETOOTH=1"],
```

Comment:
This define should carry forward from `@cyw43-driver//:cyw43_driver`. Can you check if it's safe to de-dupe this?

Reply:
I confirmed that if I eliminate this define it carries up from the driver. (still builds both with and without a btstack_config